src/main/resources/roleconfig/role_common.json

```diff
@@ -1905,7 +1905,7 @@
   {
     "name": "hearing-viewer",
     "label": "Hearing Viewer",
-    "description": "Hearing Viewer for admin users",
+    "description": "Hearing Viewer for ctsc users",
```

Comment:
Where was this change specified in the ticket? I couldn't see it...

Reply:
The change to the description wasn't specified, but I noticed the category for this role is CTSC and not ADMIN, hence why I updated the description.